lib/galaxy/jobs/__init__.py

```diff
@@ -869,7 +869,7 @@ def get_special( ):
         self.dependency_shell_commands = self.tool.build_dependency_shell_commands()
         # We need command_line persisted to the db in order for Galaxy to re-queue the job
         # if the server was stopped and restarted before the job finished
-        job.command_line = self.command_line
+        job.command_line = util.unicodify(self.command_line)
```

Comment:
`unicodify` is already explicitly imported (it's used by the `finish()` method). It'd be nice to consistently use one or the other.